sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/resource/StorageFileResource.java

```diff
@@ -197,24 +193,16 @@ StorageType getStorageType() {
         return StorageType.FILE;
     }
 
-    private void assertExisted() throws FileNotFoundException {
-        if (!exists()) {
-            throw new FileNotFoundException("File or container not existed.");
-        }
-    }
-
     private void create() throws ShareStorageException {
-        if (!shareClient.exists()) {
-            this.shareClient.create();
-        }
-        if (!shareFileClient.exists()) {
+        if (!this.exists()) {
```

Comment:
@rickle-msft @ibrahimrabab Should we introduce `shareFileClient.createIfNotExists(size)` ?